openslides_backend/action/actions/meeting/clone.py

```diff
@@ -118,28 +112,8 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             if field in instance:
                 meeting[field] = instance.pop(field)
 
-        # reset mediafile/attachment_ids to [] if None.
-        for mediafile_id in instance["meeting"].get("mediafile", []):
-            if (
-                instance["meeting"]["mediafile"][mediafile_id].get("attachment_ids")
-                is None
-            ):
-                instance["meeting"]["mediafile"][mediafile_id]["attachment_ids"] = []
-
         # check datavalidation
-        checker = Checker(
-            data=meeting_json,
-            mode="internal",
-            repair=True,
-            fields_to_remove={
-                "motion": [
-                    "origin_id",
-                    "derived_motion_ids",
-                    "all_origin_id",
-                    "all_derived_motion_ids",
-                ]
-            },
-        )
+        checker = InternalChecker(meeting_json)
```

Comment:
keep the code readable and add here the missing field `origin_meeting_id` instead complicating the code for creating a new class. Parameters are for configuring the instance

Reply:
But the code is simplified: Instead of having to manually pass all the arguments, the tasks are instead divided and `meeting.clone` does no longer need any knowledge of the parameters of the checker.